src/components/Button.jsx

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+
+const StyledButton = styled.button`
+  background: ${props =>
+    props.disabled
+      ? '#ffb6c1'
+      : props.variant === 'primary'
+      ? '#ff4d4d'
+      : props.variant === 'circle'
+      ? 'none'
+      : 'none'};
```

Comment:
have another go at this, I understand what you are trying to do but this is a bit too many ternaries to be readable. Sometimes, simpler is better. (this is an optional fix, it will not make or break your grade for this project)